src/saltext/vmware/states/vmc_org_user.py

```diff
@@ -0,0 +1,330 @@
+"""
+VMC Org User state module
+
+Add new user and delete existing user in the given org.
+
+Example usage :
+
+.. code-block:: yaml
+
+    test_user@vmware.com:
+      vmc_org_user.invite:
+        - hostname: console-stg.cloud.vmware.com
+        - refresh_key: 7jPSGSZpCa8e5Ouks4UY5cZyOtynAhF
+        - org_id: 10e1092f-51d0-473a-80f8-137652c39fd0
+        - organization_roles:
+            - name: org_member
+        - verify_ssl: False
+        - cert: /path/to/client/certificate
+
+.. warning::
+    It is recommended to pass the VMC authentication details using Pillars rather than specifying as plain text in SLS
```

Comment:
**note** with #258 nearly merged, we should be able to drastically improve these modules. I'm pretty sure we won't even need to take the hostname/refresh_key/org_id anymore, instead, the config layout will look something like this:

```
saltext.vmware:
  vmc:
    host: 203.113.0.42
    refresh_key: abcd1234
    org_id: 1234abcd
```
alternatively:
```
saltext.vmware:
  profile_1:
    vmc:
      host: 203.113.0.42
      refresh_key: abcd1234
      org_id: 1234abcd
  profile_2:
    vmc:
      host: 203.113.0.43
      refresh_key: abcd1234
      org_id: 1234abcd
```

Reply:
@waynew , we will take it as a separate PR to make change in all the modules